endpoint/verify.go

```diff
@@ -0,0 +1,100 @@
+package endpoint
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/google/uuid"
+)
+
+// Authorization is used to interact with the `jwt` package
+type Authorization struct {
+	ID      string `json:"sub"`
+	Session string `json:"https://ataper.net/session"`
```

Comment:
https://auth0.com/docs/api-auth/tutorials/adoption/scope-custom-claims

In short, nothing, it's just a prefix w add to our proprietary JWT claims to avoid collisions out there in the world. It's a convention used by a lot of identity providers, and for Auth0 in particular it's a requirement (that is, you can only edit claims that start with a non-`auth0.com` HTTP or HTTPS URL).

I'm going to change these to `jwt.ataper.net` so that these pseudo-URLs don't conflict with any real URLs we might have in the future, although it wouldn't technically matter if they did.